### ISSUE TITLE
Julia's syntax highlighter collapses Markdown code.

### ISSUE THREAD

**Issue by bicycle1885** [jupyterlab/jupyterlab#5266]
When I write this valid code snippet enclosed by triple backquotes in a Markdown cell, the syntax highlighter seems to unexpectedly collapse code between dollar marks (`$`) when rendered as HTML.
```julia
eval(:(const $(name) = $(0x01 << (i - 1))))
```

Screenshot:
<img width="355" alt="image" src="https://user-images.githubusercontent.com/905683/45068823-462add80-b104-11e8-9ada-774aa7e07388.png">

I'm using Jupyterlab 0.34.5 and IJulia 1.10.0.

**Comment by blink1073** [comment]
Hi @bicycle1885, what is the behavior in the classic notebook?

**Comment by bicycle1885** [comment]
It looks the same (see the screenshot below).
<img width="1072" alt="image" src="https://user-images.githubusercontent.com/905683/45084479-63c66a00-b139-11e8-81b2-a89230dcafd8.png">

Deleting the first or second dollar mark makes it go away. So, I think having two or more dollar marks in the same line causes the rendering problem.

I've updated some related packages but nothing has changed.
```
~/w/julia (master|…) $ jupyter --version
4.4.0
~/w/julia (master|…) $ jupyter lab --version
0.34.7
~/w/julia (master|…) $ jupyter notebook --version
5.6.0
```

**Comment by blink1073** [comment]
Thanks!  The relevant code is here:

https://github.com/jupyterlab/jupyterlab/blob/4068b094e16e01d9c9d05b55f6a199bcf4b32c39/packages/rendermime/src/latex.ts#L25

**Comment by krassowski** [comment]
This is the same issue as #10861 which was fixed by https://github.com/jupyterlab/jupyterlab/pull/11479. The fix was backported so it will be included in JupyterLab 3.2.5.

**Comment by bicycle1885** [comment]
Great!
